pkg/rpctype/rpctype.go

```diff
@@ -82,6 +82,23 @@ type RunnerConnectArgs struct {
 	Pool, VM int
 }
 
+type RunnerConnectRes struct {
+	// CheckUnsupportedCalls is set to true if the Runner needs to query the kernel
+	// for unsupported system calls and report them back to the server.
+	CheckUnsupportedCalls bool
+}
+
+// CheckUnsupportedArgs contains the date passed from client to server in a
+// CheckSupported call, namely the system calls not supported by the client's
+// kernel.
+type CheckUnsupportedArgs struct {
+	// Pool is used to identify the checked kernel.
+	Pool int
+	// Unsupported calls contains the IDs of unsupported system calls on the
+	// kernel.
+	UnsupportedCalls []int
```

Comment:
If we want to print reasons why syscalls are disabled, then we need to pass reasons here as well.